packages/core/examples/programaticPanZoom/index.ts

```diff
@@ -0,0 +1,160 @@
+import {
+  getRenderingEngine,
+  RenderingEngine,
+  Types,
+  Enums,
+} from '@cornerstonejs/core';
+import {
+  initDemo,
+  createImageIdsAndCacheMetaData,
+  setTitleAndDescription,
+  ctVoiRange,
+  addButtonToToolbar,
+} from '../../../../utils/demo/helpers';
+
+// This is for debugging purposes
+console.warn(
+  'Click on index.ts to open source code for this example --------->'
+);
+
+const { ViewportType } = Enums;
+const renderingEngineId = 'myRenderingEngine';
+const viewportId = 'CT_STACK';
+
+// ======== Set up page ======== //
+setTitleAndDescription(
+  'Programmatic Pan and Zoom with initial pan and zoom',
+  'Displays an image at the top of the viewport, half off the screen, and has pan/zoom buttons.'
```

Comment:
I tried this demo, it works fine. I'm just confused what the `reset current as initial` do actually? when I click on it it does nothing, but when I `reset original` it moves the camera. Can you add a sentence or two to say what is the expected behavior for the `reset current as initial`? is it `set current view as initial`? Can's you just trigger a render so that it actually set it? or call reset.

Reply:
Added comments - basically it allows resetting the base/original conditions, so that a subsequent reset changes to display
the current setup.